bbc/simorgh
# Add missing csp entries for includes on live

No issue

**Overall change:**
On live, VJ includes do not currently load correctly as the `src` they load their content from has been missed from our amp csp: https://www.bbc.com/tamil/science-53331648.amp 
![image](https://user-images.githubusercontent.com/9645462/87447317-565e4e80-c5f2-11ea-816d-145a0ce9f094.png)
![image](https://user-images.githubusercontent.com/9645462/87447354-6118e380-c5f2-11ea-96ba-2c1c376adf17.png)

**Code changes:**
- Update amp csp for live to add https://news.files.bbci.co.uk
- Update amp csp for test to add https://news.files.bbci.co.uk for live and https://news.test.files.bbci.co.uk for test

---

- [x] I have added labels to this PR for the relevant pod(s) affected by these changes

**Testing:**

*Ensure to use a production build so the CSP applies (`npm run build && npm run start`)*
- Visit a page such as http://localhost:7080/mundo/23263889.amp with a VJ include on observe the include loads successfully. 
- Change your [local `.env` config](https://github.com/bbc/simorgh/blob/latest/envConfig/local.env#L8) to point to the live includes amp origin (https://news.files.bbci.co.uk) to check includes load from live too.
